server/api/users.js

```diff
@@ -12,6 +13,8 @@ internals.applyRoutes = function (server, next) {
 
     const User = server.plugins['hapi-mongo-models'].User;
 
```

Comment:
Oh. Haha. I should've been more clear. This newline should be after the `Account` declaration.

Reply:
Oh, you mean an extra line? There is a newline between the declaration and `server.route`

Reply:
Yeah new blank lines before `server.route`. Sorry, I know it's a minor thing.

Reply:
Another example: https://github.com/jedireza/aqua/blob/master/server/api/signup.js#L13-L18

Reply:
Got it